reana_client/cli/workflow.py

```diff
@@ -212,12 +217,20 @@ def workflow_create(ctx, file, name, skip_validation, access_token):
 @click.option(
     '-p', '--parameter',
     multiple=True,
-    help='Optional operational parameters for the workflow execution. '
+    help='Additional input parameters to override '
+         'original ones from reana.yaml. '
+         'E.g. inputfile=new_input.txt.',
+)
+@click.option(
+    '-o', '--option',
+    multiple=True,
+    help='Optional operatioal parameters for the workflow execution. '
          'E.g. CACHE=off.',
 )
 @click.pass_context
 @with_api_client
-def workflow_start(ctx, workflow, access_token, parameter):  # noqa: D301
+def workflow_start(ctx, workflow, access_token,
+                   parameter, option):  # noqa: D301
```

Comment:
Maybe `parameters` and `options` in plural since they are specified as `multiple=True` and they are lists?